test/cpp-tableau-loader/src/protoconf/item_conf.pc.cc

```diff
@@ -202,16 +202,16 @@ const protoconf::ItemConf::Item* ItemConf::FindFirstItemExtInfo(protoconf::Fruit
 // Index: (ID,Name)<Type,UseEffectType>@AwardItem
 const ItemConf::Index_AwardItemMap& ItemConf::FindAwardItem() const { return index_award_item_map_ ;}
```

Comment:
Naming consistence.
Just like Go, we should rename the API when return the underlying hash map of index: `FindAwardItem` -> `FindAwardItemMap`.